Aries/Aries/Admin/Admin.swift

```diff
@@ -15,17 +15,21 @@ public class Admin {
     private var storage: Storage
     private var events: AriesEvents
     private var agentConnections: AriesConnections
+    private var agentProofs: AdminProofs
+    private var agentCredentials: AdminCredentials
     private var adminInvitationUrl: String?
     public var connections: AdminConnections? = nil
     
     //connectToAdmin data for reference in event handling... there might be a better way to do this?
     private var adminConnectionId: String? = nil
     private var adminCompletion: (_ result: Result<Void, Error>)->Void
     
-    public init(messageSender: MessageSender, storage: Storage, connections: AriesConnections, events: AriesEvents, adminInvitationUrl: String? = nil){
+    public init(messageSender: MessageSender, storage: Storage, connections: AriesConnections, proofs: AdminProofs, credentials: AdminCredentials, events: AriesEvents, adminInvitationUrl: String? = nil){
         self.messageSender = messageSender
         self.storage = storage
         self.agentConnections = connections
+        self.agentProofs = proofs
+        self.agentCredentials = credentials
```

Comment:
```suggestion
```
submodules are not constructed at class creation but after the adminConnection is set in setAdminConnection